### ISSUE TITLE
pytest fails for me

### ISSUE THREAD

**Issue by ahbarnett** [fruzsinaagocs/riccati#9]
After a pip install of riccati and pytest, on py 3.10, following the instructions, it doesn't find the riccati/test directory so no tests are run:

```
(py310) alex@ross /home/alex/physics/agocs> python -m pip install -U pytest
python -m pytest -v riccati/tests
Collecting pytest
  Downloading pytest-7.2.1-py3-none-any.whl (317 kB)
     |████████████████████████████████| 317 kB 958 kB/s 
Collecting tomli>=1.0.0
  Using cached tomli-2.0.1-py3-none-any.whl (12 kB)
Collecting exceptiongroup>=1.0.0rc8
  Downloading exceptiongroup-1.1.0-py3-none-any.whl (14 kB)
Requirement already satisfied: packaging in /home/alex/miniconda3/envs/py310/lib/python3.10/site-packages (from pytest) (21.3)
Collecting pluggy<2.0,>=0.12
  Using cached pluggy-1.0.0-py2.py3-none-any.whl (13 kB)
Collecting attrs>=19.2.0
  Downloading attrs-22.2.0-py3-none-any.whl (60 kB)
     |████████████████████████████████| 60 kB 916 kB/s 
Collecting iniconfig
  Downloading iniconfig-2.0.0-py3-none-any.whl (5.9 kB)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/alex/miniconda3/envs/py310/lib/python3.10/site-packages (from packaging->pytest) (3.0.7)
Installing collected packages: tomli, pluggy, iniconfig, exceptiongroup, attrs, pytest
Successfully installed attrs-22.2.0 exceptiongroup-1.1.0 iniconfig-2.0.0 pluggy-1.0.0 pytest-7.2.1 tomli-2.0.1
======================================= test session starts ========================================
platform linux -- Python 3.10.0, pytest-7.2.1, pluggy-1.0.0 -- /home/alex/miniconda3/envs/py310/bin/python
cachedir: .pytest_cache
rootdir: /home/alex/physics/agocs
collected 0 items                                                                                  

====================================== no tests ran in 0.00s =======================================
ERROR: file or directory not found: riccati/tests
```

** Versions (please complete the following information):**
 - OS:   ubuntu 22.04
 - Python, numpy, riccati version

See above.
```
>>> np.__version__
'1.21.5'
```

riccati 1.0.1

**Additional context**

**Comment by fruzsinaagocs** [comment]
This will help me diagnose the issue: did you install via pip or from source?

**Comment by fruzsinaagocs** [comment]
Here's how the unit tests can be run currently:
- they can be run manually like you did above if one installs from source, i.e. `git clone ... ; pip install .; python -m pytest riccati/tests` -> this is the advice that appears in the documentation. 
- they will run automatically if you `conda install` as part of the installation recipe (but I haven't seen the equivalent happen during pip installs of prominent packages).
- they also run on every push and PR.

So, if you used `pip` to install, you'd need to `cd` into wherever the source is to run the tests.

**Comment by ahbarnett** [comment]
We just need the docs to be a sequence the user can run successfully, which they are not right now, since it says first pip-install, then run the pytest (which fails).

So, best would be how to pytest given you did pip install.
I tried
cd /home/alex/miniconda3/envs/py310/lib/python3.10/site-packages/riccati
but then
python -m pytest -v riccati/tests
fails.
I also tried from the parent.

I didn't try git clone install yet...  Just trying to get the docs so the instructions work!
